homeassistant/components/fritz/common.py

```diff
@@ -86,7 +86,6 @@ def __init__(
         self._unique_id: str | None = None
         self.connection: FritzConnection = None
```

Comment:
The typing on self._devices could likely be

`        self._devices: dict[str, FritzDevice] = {}`

It too far away to allow github to comment though

Reply:
Thanks. Applied it.